popde/adaptive_kde.py

```diff
@@ -250,15 +251,19 @@ class KDERescaleOptimization(AdaptiveBwKDE):
     """
     def __init__(self, data, weights=None, input_transf=None,
                  stdize=False, rescale=None, backend='KDEpy', bandwidth=1.0, alpha=0.5,
-                 dim_names=None, do_fit=False, n_splits=5):
+                 dim_names=None, do_fit=False, n_splits=5, uniform_rescale=False, uniform_dims=None):
```

Comment:
why do you need two extra arguments here?  I think you could just supply uniform_dims and if this is None do nothing.
Also is there any reason why uniform_dims should have exactly 2 dimensions, could it be used for any number of dimensions?